docs/source/recipes/qat_distributed.rst

```diff
@@ -0,0 +1,88 @@
+.. _qat_distributed_recipe_label:
+
+=============================================
+Distributed Quantization-Aware Training (QAT)
+=============================================
+
+QAT allows for taking advantage of memory-saving optimizations from quantization at inference time, without significantly
+degrading model performance. In torchtune, we use `torchao <https://github.com/pytorch/ao>`_ to implement QAT during training.
+This works by :ref:`simulating quantization numerics during fine-tuning <what_is_qat_label>`. While this may introduce memory and
+compute overheads during training, our tests found that QAT significantly reduced performance degradation in evaluations of
+quantized model, without compromising on model size reduction gains.
```

Comment:
You can also point to this recent blog: https://pytorch.org/blog/quantization-aware-training/

Reply:
nice blog